astro-docs/src/content/docs/concepts/Module Federation/faster-builds.mdoc

```diff
@@ -175,13 +181,63 @@ the `implicitDependencies` configuration.
 
 In the future, Nx may automatically handle this for you.
 
-### `apps/host/webpack.config.ts`
+### React: `apps/host/rspack.config.js`
+
+With React, we use Rspack by default and generate a compliant Rspack configuration file:
+
+```javascript
+import { NxAppRspackPlugin } from '@nx/rspack/app-plugin.js';
+import { NxReactRspackPlugin } from '@nx/rspack/react-plugin.js';
+import {
+   NxModuleFederationPlugin,
+   NxModuleFederationDevServerPlugin,
+} from '@nx/module-federation/rspack.js';
+import { join } from 'path';
+
+import config from './module-federation.config';
```

Comment:
nit: I noticed we import with extensions above for `@nx/rspack` projects. should we remain consistent and import the config also with extension? I know it'll work either way. just wanted to call it out